### DISCUSSION TITLE
Pod name as shown in Argo CLI does not match with actual pod name

### DISCUSSION THREAD

**Discussion by andyshieh** [argoproj/argo-workflows#7630] [Q&A] [answered]
Dear Argo community,
I am currently using 3.2.6 for argoexec, argocli, workflow-controller, and argo cli (linux amd64).
The actual pod names produced by Argo are [workflow id]-[task template]-[random numeric string] which is what I want.
When using `kubectl get pods` I also see the actual pod names.
However, when using `argo watch` the pod names are missing the [task template] part and is only showing [workflow id]-[random numeric string]
Has anyone encountered similar issues after v2 pod name changes? Am I missing certain configurations or is this a bug?
Thank you for your help.

**Comment by terrytangyuan** [comment, marked as answer]
This seems like a bug due to pod name change. Could you file a bug report instead and provide additional debugging information?

**Reply by terrytangyuan** [reply]
cc @JPZ13

**Reply by JPZ13** [reply]
Thanks @terrytangyuan!

**Comment by JPZ13** [comment]
Hi @andyshieh - this is most likely a bug. Could you raise an issue and assign it to me? I'll get working on it

**Reply by andyshieh** [reply]
Thanks @JPZ13 and @terrytangyuan . I've created an issue here and tagged @JPZ13 (don't think I can assign; or perhaps I'm missing something)
https://github.com/argoproj/argo-workflows/issues/7646
